docs/data-sources/gateway.md

```diff
@@ -0,0 +1,236 @@
+
+Read-Only:
+
+- `id` (String)
+
+
+
+
+<a id="nestedatt--aws_fargate_task_definition--task_role"></a>
+### Nested Schema for `aws_fargate_task_definition.task_role`
+
+Read-Only:
+
+- `arn` (String)
+
+
+
+<a id="nestedatt--cluster_ip"></a>
+### Nested Schema for `cluster_ip`
+
+Read-Only:
+
+- `name` (String)
+
+
+<a id="nestedatt--gcp_cloud_run"></a>
+### Nested Schema for `gcp_cloud_run`
+
+Read-Only:
+
+- `id` (String)
+- `serverless_vpc_connector` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_run--serverless_vpc_connector))
+- `service_account` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_run--service_account))
+
+<a id="nestedatt--gcp_cloud_run--serverless_vpc_connector"></a>
+### Nested Schema for `gcp_cloud_run.serverless_vpc_connector`
+
+Read-Only:
+
+- `id` (String)
+
+
+<a id="nestedatt--gcp_cloud_run--service_account"></a>
+### Nested Schema for `gcp_cloud_run.service_account`
+
+Read-Only:
+
+- `self_link` (String)
+
+
+
+<a id="nestedatt--k8s_container"></a>
+### Nested Schema for `k8s_container`
+
+Read-Only:
+
+- `deployment` (Attributes) (see [below for nested schema](#nestedatt--k8s_container--deployment))
+- `name` (String)
+
+<a id="nestedatt--k8s_container--deployment"></a>
+### Nested Schema for `k8s_container.deployment`
+
+Read-Only:
+
+- `name` (String)
+- `namespace` (Attributes) (see [below for nested schema](#nestedatt--k8s_container--deployment--namespace))
+- `service_account` (Attributes) (see [below for nested schema](#nestedatt--k8s_container--deployment--service_account))
+
+<a id="nestedatt--k8s_container--deployment--namespace"></a>
+### Nested Schema for `k8s_container.deployment.namespace`
+
+Read-Only:
+
+- `aws_eks` (Attributes) (see [below for nested schema](#nestedatt--k8s_container--deployment--namespace--aws_eks))
+- `gcp_gke` (Attributes) (see [below for nested schema](#nestedatt--k8s_container--deployment--namespace--gcp_gke))
+- `name` (String)
+
+<a id="nestedatt--k8s_container--deployment--namespace--aws_eks"></a>
+### Nested Schema for `k8s_container.deployment.namespace.name`
+
+Read-Only:
+
+- `arn` (String)
+
+
+<a id="nestedatt--k8s_container--deployment--namespace--gcp_gke"></a>
+### Nested Schema for `k8s_container.deployment.namespace.name`
+
+Read-Only:
+
+- `id` (String)
+
+
+
+<a id="nestedatt--k8s_container--deployment--service_account"></a>
+### Nested Schema for `k8s_container.deployment.service_account`
+
+Read-Only:
+
+- `aws_role` (Attributes) (see [below for nested schema](#nestedatt--k8s_container--deployment--service_account--aws_role))
+- `gcp_service_account` (Attributes) (see [below for nested schema](#nestedatt--k8s_container--deployment--service_account--gcp_service_account))
+- `name` (String)
+
+<a id="nestedatt--k8s_container--deployment--service_account--aws_role"></a>
+### Nested Schema for `k8s_container.deployment.service_account.name`
+
+Read-Only:
+
+- `arn` (String)
```

Comment:
It's unexpected to see an `arn` for a service account name. Actually I'm confused as to what's going on here, since there's a self-link below

Reply:
I think this is a bug in the docs generation. If you look at the id of the link which has been generated it should be called service_account.aws_role. Haven't had time to investigate, but I don't believe it's my bug